### ISSUE TITLE
"IMAGES" tag also shows intermediate images.

### ISSUE THREAD

**Issue by ikedaj** [shipyard/shipyard#667]
I got the latest shipyard image from DockerHub.
It worked fine, but I found a strange view at "IMAGES" tag.
It shows "docker images -a", it means we can also see the intermediate images.
for example, "&lt;none>:&lt;none>".

get('/images/json?all=0') ?
https://github.com/shipyard/shipyard/blob/master/controller/static/app/images/images.service.js#L13

But it's really strange, when I tried Shipyard v3.0.0 with Docker 1,7, 
images tag just showed the expected result... 

Here is my setup;

* Docker Version

```
# docker -v

Docker version 1.8.2, build bb472f0/1.8.2
```

* Get the latest images

```
# docker pull docker.io/swarm
# docker pull docker.io/rethinkdb
# docker pull docker.io/shipyard/shipyard
```

* Get the latest Docker Compose

```
# curl -L https://github.com/docker/compose/releases/download/1.5.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
# chmod +x /usr/local/bin/docker-compose
```

* Create compose.yml

```
# vim compose-shipyard.yml

shipyard:
  container_name: shipyard
  image: docker.io/shipyard/shipyard
  restart: always
  links:
   - shipyard-swarm:swarm
   - shipyard-rethinkdb:rethinkdb
  ports:
   - 8080:8080
  environment:
   - "HTTP_PROXY=http://USERNAME:PASSWORD@PROXY:PORT/"
   - "HTTPS_PROXY=http://USERNAME:PASSWORD@PROXY:PORT/"
   - "NO_PROXY=172.26.210.200"
  command: server --rethinkdb-addr=rethinkdb:28015 --disable-usage-info --docker tcp://172.26.210.200:2377

shipyard-rethinkdb:
  container_name: shipyard-rethinkdb
  image: docker.io/rethinkdb
  restart: always
  command: rethinkdb --bind all --no-update-check

shipyard-swarm:
  container_name: shipyard-swarm
  image: docker.io/swarm
  restart: always
  ports:
   - 2377:2375
  command: manage nodes://172.26.210.200:2375
```

* Startup containers

```
# docker-compose -f compose-shipyard.yml up -d
```

* Access to Shipyard

```
http://172.26.210.200:8080
```

**Comment by ikedaj** [comment]
Is this expected?

![images](https://cloud.githubusercontent.com/assets/724236/11391798/9a95233a-9398-11e5-8240-7042d5c81e2b.PNG)

**Comment by ehazlett** [comment]
Fixed in #682
